sdk/go/pulumi/types.go

```diff
@@ -620,6 +620,24 @@ func JSONMarshalWithContext(ctx context.Context, v interface{}) StringOutput {
 	}).(StringOutput)
 }
 
+// JSONUnmarshal uses "encoding/json".Unmarshal to deserialize the given Input JSON string into a value.
+func JSONUnmarshal(v StringInput) AnyOutput {
+	return JSONUnmarshalWithContext(context.Background(), v)
+}
+
+// JSONUnmarshalWithContext uses "encoding/json".Unmarshal to deserialize the given Input JSON string into a value.
+func JSONUnmarshalWithContext(ctx context.Context, data StringInput) AnyOutput {
```

Comment:
Nit: Use the same `v` parameter name for both "overloads"?

```suggestion
func JSONUnmarshalWithContext(ctx context.Context, v StringInput) AnyOutput {
```

Reply:
Made them both "data" to match the parameter name of json.Unmarshal